narwhal/network/src/connectivity.rs

```diff
@@ -68,10 +68,6 @@ impl ConnectionMonitor {
                 .set(0)
         }
 
-        self.connection_metrics
-            .network_peers
-            .set(connected_peers.len() as i64);
```

Comment:
Does items in `connected_peers` ever get cleaned up?

Reply:
`connected_peers` is quite ephemeral here - it is just to update the metrics once we subscribe to the network updates.